tests/file_storage/tests.py

```diff
@@ -807,6 +810,9 @@ def test_callable_storage_param(self):
         obj = Storage.objects.create()
         self.assertIs(obj.callable_storage.storage, temp_valid_name_storage)
 
+    def test_callable_base_class_error_raises(self):
+        self.assertRaises(ImproperlyConfigured, FileField(storage=lambda x:x))
```

Comment:
* Please use `assertRaisesMessage` here. 
* `flake8` will require whitespace after the `:` in the lambda.

Reply:
All done, tests running now